SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/StringStorage.h

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
```

Comment:
Nit spacing - probably need a blank line here and above the each constructor / member function.